lib/src/util/class_names.dart

```diff
@@ -150,6 +166,32 @@ class ClassNameBuilder {
     _blacklistBuffer.write(className);
   }
 
+
+
+  /// Merges together the classes and blacklists present in another `ClassNameBuilder` instance.
+  /// 
+  /// Takes [other._blacklistBuffer] and merges it into [_blacklistBuffer]
+  /// and takes [other._classNamesBuffer] and merges it into [_classNamesBuffer].
+  void merge(ClassNameBuilder other) {
+    if (_blacklistBuffer == null) {
+      _blacklistBuffer = new StringBuffer();
+    } else {
+      if (_blacklistBuffer.isNotEmpty) {
+        _blacklistBuffer.write(' ');
+      }
+    }
+    _blacklistBuffer.write(other._blacklistBuffer);
+
+    if (_classNamesBuffer == null) {
+      _classNamesBuffer = new StringBuffer();
```

Comment:
This is always initialized (resulting in this line missing test coverage), so this case can probably be removed.